ads/amoad.js

```diff
@@ -21,13 +21,26 @@ import {loadScript, checkData, validateDataExists} from '../3p/3p';
  * @param {!Object} data
  */
 export function amoad(global, data) {
-  checkData(data, ['sid']);
+  checkData(data, ['sid', 'adType']);
   validateDataExists(data, ['sid']);
```

Comment:
Added the attribute 'data-ad-type' as optional. supposed to be accessed via 'adType', right?